src/ToDoBody/ToDoItem.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { View, Text, StyleSheet, TouchableOpacity } from 'react-native';
+
+function ToDoItem(props) {
+    return (
+        <TouchableOpacity onPress = {props.onDelete.bind(this, props.id)}>
```

Comment:
```suggestion
        <TouchableOpacity onPress = {() => props.onDelete(props.id)}>
```

You don't use the 'this' keyword in the function anymore, so it doesn't need to be used now. Cool that you knew how to do that though! :)